spec/rubocop/cop/rspec/described_class_spec.rb

```diff
@@ -24,6 +24,26 @@
         end
       RUBY
     end
+
+    it 'ignores offenses within non-rspec numblocks' do
+      expect_offense(<<~RUBY)
+        describe MyClass do
+          controller(ApplicationController) do
+            _1
```

Comment:
What is the real-world use case for this?

Reply:
This is just here to make the `block` node into a `numblock` node. Since this is a user-defined method it can yield as much or as little as it wants and the user can use a few ways to express that. It just aligns it to work for all types of blocks there are (currently `foo {}`, `foo { |bar| }` and `foo { _1 }`, but also `foo { it }` in the future from Ruby 3.4).

I made the spec a bit more aligned to how could look like in actual code.